app/app.go

```diff
@@ -723,6 +723,7 @@ func New(
 
 	app.mm.SetOrderPreBlockers(
 		upgradetypes.ModuleName,
+		authtypes.ModuleName,
```

Comment:
Consider adding this to the modules.go , similar to how we have listed order for endblocker and initblocker